config/default/manager/manager.yaml

```diff
@@ -42,6 +42,8 @@ spec:
         app: kubemacpool
         control-plane: mac-controller-manager
         controller-tools.k8s.io: "1.0"
+      annotations:
+        description: KubeMacPool manager pod manages MAC allocation to Pods and VMs in Kubernetes cluster
```

Comment:
No need to mention "Kubernetes cluster", you already are in one, talking about Pods and VMs.

Also, can we keep it short without too much managing? KubeMacPool managers MAC allocation

Reply:
[Done](https://github.com/k8snetworkplumbingwg/kubemacpool/compare/e52d4def2e4ff6f26b5c263f1f491ad5a95b677f..b733307496f72e3e1cfd4cb7e9b68a3354202e20)